tests/main/test_sciphi_search_rag_tool.py

```diff
@@ -0,0 +1,129 @@
+"""
+NOTE: running this example requires setting the SCIPHI_API_KEY environment variable in your `.env` file, as explained in the documentation for the SciPhi API.
+
+"""
+import itertools
+import pytest
+
+from langroid.agent.chat_agent import ChatAgent, ChatAgentConfig
```

Comment:
We can make this test more focused on this search tool, so you can get rid of the other aspects. I will do the same for the `test_google_search_tool.py`.

Reply:
If you can do w/ the google search tool i will follow suit

Reply:
I pushed a simplified test of the google search. I removed 2 tests and their associated setup/constants.